app/src/main/java/com/nervousfish/nervousfish/service_locator/NervousFish.java

```diff
@@ -0,0 +1,79 @@
+package com.nervousfish.nervousfish.service_locator;
+
+import android.app.Application;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.IBinder;
+
+import com.nervousfish.nervousfish.modules.pairing.AndroidBluetoothService;
+import com.nervousfish.nervousfish.modules.pairing.PairingWrapper;
+
+/**
+ * The controller of the NervousFish application
+ */
+public final class NervousFish extends Application implements INervousFish {
```

Comment:
In what ways does this differ from the `EntryActivity`/Why can't this be done by the `EntryActivity`?